tests/tiny_arena.rs

```diff
@@ -0,0 +1,26 @@
+
+#[cfg(not(miri))] // this will take too much time for miri
+#[test]
+fn add_65535_objects() {
+    use compact_arena::in_tiny_arena;
+
+    in_tiny_arena!(arena, {
+        for _ in 0..65535 {
```

Comment:
Like with `NanoArena`, `TinyArena' now can hold 65536 elements and can be created with `mk_tiny_arena!(arena);`. Please update the test accordingly.